packages/tools/devtools/devtools-core/src/FluidDevtools.ts

```diff
@@ -174,11 +193,16 @@ export class FluidDevtools implements IFluidDevtools {
 	 */
 	private readonly postSupportedFeatures = (): void => {
 		const supportedFeatures = this.getSupportedFeatures();
+		let unsampledTelemetry = sessionStorage.getItem(unsampledTelemetryKey) === "true";
+		if (unsampledTelemetry === null) {
+			unsampledTelemetry = false;
+		}
```

Comment:
I think the `if` is unnecessary (and `unsampledTelemetry` can become `const`) because the line above that will always return a boolean. Even if the value in sessionStorage is null, the variable is set to the result of comparing `<whatever was in sessionStorage> === "true"`, and that can only ever be true or false.

In fact I'm a bit surprised Typescript didn't complain about something here... I would have expected it to infer the type of `unsampledTelemetry` as boolean and then say "a boolean will never equal null, this `if` will never apply", but I might be missing something there.